index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Jonathan Arias Gallego | Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <nav id="navbar" class="open">
+      <div class="navbar__status">
+        <img src="Assets/Time Style.svg" alt="">
+        <img src="Assets/Bars.svg" alt="">
+      </div>
```

Comment:
- [ ] [OPTIONAL] Although this is a bit confusing the icons above the navbar are only shown in the design to create a better mobile simulation and shouldn't be present in the actual implementation of your design so I highly advise you remove it so that it doesn't confuse users.
- Here's how it looks:
  ![image](https://github.com/sinruidotec/Portfolio-setup-mobile-first/assets/104968236/4b47d509-8a8f-4813-8aa2-61acd4defe37)